src/interfaces/highs_c_api.h

```diff
@@ -14,9 +14,74 @@
 #define HIGHS_C_API
 
 #include "util/HighsInt.h"
-const HighsInt HighsStatuskError = -1;
-const HighsInt HighsStatuskOk = 0;
-const HighsInt HighsStatuskWarning = 1;
+
+const HighsInt kHighsStatusError = -1;
```

Comment:
Two comments:
 * This is technically breaking for anyone who was using this in the C API
 * Why the `k` prefix? Is that standard somewhere? 

I had it as `[enum class][enum name]` which seemed to make sense to me. The class is `HighsStatus` and the enum is `kError`. It also makes all of the symbols exported by HiGHS in the C interface start with `Highs` which makes discoverability easier. But I'm not super attached to it, so `kHighs` is okay if you prefer.